examples/chip-tool/commands/parse/Commands.h

```diff
@@ -0,0 +1,46 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "ManualCodeCommand.h"
+#include "QRCodeCommand.h"
+
+class ManualCodeParse : public ManualCodeCommand
+{
+public:
+    ManualCodeParse() : ManualCodeCommand("manual") {}
+};
+
+class QRCodeParse : public QRCodeCommand
+{
+public:
+    QRCodeParse() : QRCodeCommand("qr") {}
+};
+
+void registerCommandsParse(Commands & commands)
+{
+    const char * clusterName = "Parse";
```

Comment:
In case we want to do more with chip-tool than just parsing the initial payload. Can you rename "Parse" -> "Payload"  and rename the folders accordingly ?

Then the command to parse a payload would be: `chip-tool payload parse something`. If we ever end up adding the capabilities to generate a payload we would be able to add a `chip-tool payload generate something` command.

Reply:
Great point, the command modified to support `chip-tool payload parse something`